training/distributed_training/tensorflow/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    print(f"SageMaker Execution Role:{role}")
</pre><p>Nit - add space after colon for print statements: <code>: {</code> (3 occurrences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>